torch/_dynamo/convert_frame.py

```diff
@@ -696,6 +698,18 @@ def count_args(code):
             UncapturedHigherOrderOpError,
             BisectValidationException,
         ) as e:
+            if isinstance(e, Unsupported):
+                # Log this message in graph break logger because this can happen
+                # when we dont support a particular bytecode like LOAD_ATTR.
+                if graph_break_log.isEnabledFor(logging.DEBUG):
+                    user_stack = e.real_stack
+                    user_stack_formatted = "".join(traceback.format_list(user_stack))
+                    graph_break_log.debug(
+                        "Graph break: skip: from user code at:\n%s",
+                        user_stack_formatted,
+                        exc_info=True,
+                    )
```

Comment:
I feel this is not the right place to do the catch. We are still propagating the exception upwards. The graph break log should happen when we suppress the exception.

Reply:
The trace_id is not there available there. Its outside the _compile region.

Reply:
This is the place where we propagate the exception upwards. I tried to put the log here, but the trace_id is not available there. So, we dont see prefixes like [0/0]

https://github.com/pytorch/pytorch/blob/852111e1c20c38eb260b3975ff7d90e511e7d93f/torch/_dynamo/convert_frame.py#L807-L841

Reply:
Sorry for the spam - But I took your suggestion. Just stashing the compile_id to be used later on.

Reply:
My counterproposal is that the error suppression should be moved inwards until the point where tracing id is set, OR the setting of tracing id is moved outwards so that it is coincident with this.

But I'm also... skeptical that this is the true Unsupported handler block? If it was, shouldn't you have gotten an INFO log for the exception anyways?

Reply:
It was on info. But if you search for graph_break, you won't find it.

Reply:
Too much logging :P

Reply:
There is definitely duplication. I feel a usual reason of perf tanking is excessive graph break. So, just searching for `__graph_break` should give a good indication of what happened between a good and bad run.

I understand we can search for `WON'T CONVERT` in addition to `__graph_break` as a habit.

Hmm ... lets discuss offline.